lib/susedistribution.pm

```diff
@@ -168,5 +166,55 @@ sub become_root() {
     send_key('ctrl-l');
 }
 
+# initialize the consoles needed during our tests
+sub activate_consoles {
+    my ($self) = @_;
+    if (get_var('BACKEND', 'qemu')) {
+        activate_console('install-shell',    'tty-console', {tty => 2});
+        activate_console('root-console',     'tty-console', {tty => 2});
+        activate_console('bernhard-console', 'tty-console', {tty => 4});
```

Comment:
This is clever, indeed, but I would not mix the console name and user name. I would rather use that as an option.